languages/elixir/exercises/concept/regular-expressions/.docs/instructions.md

```diff
@@ -0,0 +1,77 @@
+You have been tasked to write a service which ingests events. Each event has a date associated with it, but you notice that 3 different formats are being submitted to your service's endpoint:
```

Comment:
I had never considered returning the regular expressions, but I think it is a really elegant solution!